tests/parser.cpp

```diff
@@ -2479,20 +2468,20 @@ k:f { print(5e-9); }
       ~~~~~~~~
 )");
 
-  test_parse_failure("k:f { print(1e17); }", R"(
-stdin:1:7-17: ERROR: Exponent will overflow integer range: 17
-k:f { print(1e17); }
+  test_parse_failure("k:f { print(1e21); }", R"(
+stdin:1:7-17: ERROR: overflow error, maximum value is 18446744073709551615: 1e21
```

Comment:
nit:
```suggestion
stdin:1:7-17: ERROR: overflow error, maximum value is 18446744073709551615. Found: 1e21
```

Reply:
Would still be nice to fix this but it's not a blocker.

Reply:
It's tough to frame chained errors in this way, i.e. it assumes that you can return an error with multiple complete sentences (versus a sentence fragment that someone else can chain). I will submit as is, and we can leave the broader question of clean errors.